test/calculate-svg-dimensions.test.jest.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const { readFileSync } = require('fs');
+const path = require('path');
+const calculateSvgDimensions = require('../lib/svg-sprite/utils/calculate-svg-dimensions.js');
+
+describe('calculateSvgDimensions', () => {
+    it('should return the expected dimensions from 46x46 fixture', () => {
+        expect.hasAssertions();
```

Comment:
This is for the eslint-plugin-jest. The point is if the test has no assertions at all it will be marked as passed no matter what it does nothing. It added false confidence in tests. 
This eslint rule guards for such errors